browser/speedreader/sources.gni

```diff
@@ -24,4 +24,21 @@ if (enable_speedreader) {
     "//components/keyed_service/core",
     "//content/public/browser",
   ]
+
+  if (is_android) {
+    brave_browser_speedreader_sources += [
+      "//brave/browser/android/speedreader/speedreader_utils.cc",
+      "//brave/browser/android/speedreader/speedreader_utils.h",
+    ]
+
+    brave_browser_speedreader_deps += [
+      "//base",
```

Comment:
remove duplicated deps as they are included above for desktop already
`//base`, ...

Reply:
Fixed.